Lesson22/selcan/app.js

```diff
@@ -0,0 +1,47 @@
+const formElement = document.getElementById('toDoForm');
+const formInput = document.getElementById('toDoInput');
+const list = document.getElementById('toDoList');
+
+formElement.addEventListener('submit', function (event) {
+	// By default, the page is reloded when form is submitted. To avoid it, we need to add `preventDetault()`;
+	event.preventDefault();
+
+  // TODO: check that input value has correct format (e.g. minimum 3 chars, no extra spaces)
+
+	const listItem = document.createElement('li');
+
+	const button = document.createElement('button');
+	button.innerText = 'x';
+	button.classList.add('button--light');
+
+	
+
+	const span = document.createElement('span');
+	span.innerText = formInput.value;
```

Comment:
Don't assign the value from the input to span element before checking if it's correctly formatted (lines 21-25). So you can move `span.innerText = formInput.value;` after the if statement